data_model/derive/src/filter.rs

```diff
@@ -200,14 +200,13 @@ pub fn impl_filter(event: &EventEnum) -> TokenStream {
     let filter_doc = format!(" Filter for {event_ident} entity");
 
     quote! {
-        iroha_data_model_derive::model! {
```

Comment:
what is the reason to remove this?

Reply:
You're right, there should be `model_single!`